rmw/include/rmw/rmw.h

```diff
@@ -188,6 +188,24 @@ RMW_WARN_UNUSED
 rmw_ret_t
 rmw_publish(const rmw_publisher_t * publisher, const void * ros_message);
 
+/// Retrieve the number of matched subscriptions to a publisher
```

Comment:
```suggestion
/// Retrieve the number of matched subscriptions to a publisher.
```

Seems like the rest of the file uses punctuation on the brief.